tests/details/details-page.spec.ts

```diff
@@ -70,20 +70,34 @@ test(`Load theater: image (single)`, async ({ detailsPage }) => {
   await detailsPage.imageCarouselMultipleImageDisplay(false);
 });
 
-test(`Load theater: radio`, async ({ detailsPage }) => {
-  await detailsPage.gotoPage('WGBH_89_7_FM_20210918_040000');
-  await detailsPage.radioPlayerTheaterDisplay();
+test(`Load theater: radio as priv'd user`, async ({
+  loginPage,
+  detailsPage,
+}) => {
+  await test.step(`Do login as priv'd user`, async () => {
+    await loginPage.loginAs('privs');
+  });
+  await test.step(`Go to radio details page and verify priv'd user borrow program`, async () => {
```

Comment:
@dualcnhq, you have tested this with privs user login, we should get it tested with patrol login or guest user so that we can make sure that button is appear in specific conditions (eg. logged-in with priv's).

Every test should be success and failure case to make sure that we things are working expected with different userStats. Thanks!

Reply:
thanks for pointing that out, i just updated the test and added those.